cards/multilang-product-prominentvideo/component.js

```diff
@@ -0,0 +1,77 @@
+{{> cards/card_component componentName='multilang-product-prominentvideo' }}
+
+class multilang_product_prominentvideoCardComponent extends BaseCard['multilang-product-prominentvideo'] {
+  constructor(config = {}, systemConfig = {}) {
+    super(config, systemConfig);
+  }
+
+  /**
+   * This returns an object that will be called `card`
+   * in the template. Put all mapping logic here.
+   *
+   * @param profile profile of the entity in the card
+   */
+  dataForRender(profile) {    
+    profile.featuredMessage = {
+      description: 'subtitle'
+    }
+    
+    const videoUrl = profile.videos && profile.videos[0]?.video?.url;
```

Comment:
Will the data stored in `videos` always be Youtube URLs? We'll never have Vimeo videos in this field?

Reply:
checking with product!

Reply:
ya, these are just youtube videos